flexmeasures/data/migrations/versions/6cf5b241b85f_copy_attributes_from_old_data_models_to_GenericAsset.py

```diff
@@ -279,13 +293,44 @@ def upgrade():
     copy_sensor_columns(connection, t_market, t_sensor)
     copy_sensor_columns(connection, t_weather_sensor, t_sensor)
     copy_sensor_columns(connection, t_asset, t_sensor)
+    copy_location_columns(connection, t_weather_sensor, t_generic_asset, t_sensor)
```

Comment:
I might be missing something but why did you create `copy_location_columns` ― it seems you are calling `copy_attributes` to get the job done.

Reply:
I have seen in practice some assets, all part of the same house, being located a few meters apart.

`copy_location_columns` copies to GenericAsset's latitude and longitude columns. Since several Assets may become part of the same GenericAsset (when they become Sensors), location info may be lost (in case not all of the sensors share the exact same location). I have no preference from which of the sensors we copy the location info.

I call `copy_attributes` to also copy the location info to each Sensor's attributes column. That way, no information is lost.